enclave_wrangler/objects_api.py

```diff
@@ -793,22 +805,23 @@ def refresh_tables_for_object():
 
 
 def get_concept_set_version_expression_items(
-    version_id: Union[str, int], return_detail, handle_paginated=True
+    version_id: Union[str, int], return_detail, handle_paginated=True, fail_on_error=True
 ) -> List[Dict]:
     """Get concept set version expression items"""
     version_id = str(version_id)
     items: List[Dict] = get_object_links(
         object_type='OMOPConceptSet',
         object_id=version_id,
         link_type='OmopConceptSetVersionItem',
-        handle_paginated=handle_paginated)
+        handle_paginated=handle_paginated,
```

Comment:
The counts tables won't be current enough on the enclave to help, probably.

Getting the member count before getting the members is a good question -- probably for Amin. I don't know a way.

Since the new test cset versions I made yesterday and today have more than 50K members, the objects API won't work anyway -- so the test whether the object api approach and dataset api approach will yield the same results will at least fail on those two version.

Reply:
Actually, I was able to fetch more than 50k members. But I wasn't able to fetch more than 10k expression items (which somehow appeared to be the case for these sets)